src/System Application/App/Page Summary Provider/src/PageSummaryProviderImpl.Codeunit.al

```diff
@@ -106,7 +113,17 @@ codeunit 2717 "Page Summary Provider Impl."
         exit(Format(ResultJsonObject));
     end;
 
-    procedure GetFieldsSummary(PageId: Integer; RecId: RecordId; Bookmark: Text; var ResultJsonObject: JsonObject): Text
+    local procedure AddFields(PageId: Integer; RecId: RecordId; Bookmark: Text; var ResultJsonObject: JsonObject)
+    begin
+        // Add Summary fields
+        GetFieldsSummary(PageId, RecId, Bookmark, ResultJsonObject);
+
+        // Add Record fields
+        if not ResultJsonObject.Contains('error') then
+            GetRecordFields(PageId, Bookmark, ResultJsonObject);
```

Comment:
To me it looks like `GetFieldsSummary` and `GetRecordFields` do the same thing more or less, the only difference being that one of them is `pfields` and another is `tfields`,
Wouldn't it be more suitable to call it  `GetRecordFieldsSummary` then ?